scripts/semver.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+###
+# semantic version comparition using semver specification http://semver.org/
```

Comment:
I believe that `comparition` is misspelled `comparison`. Still, I think it might sound better when re-worded.

Reply:
Hi @tukusejssirs,

For this and the comments below in respect to `semver.sh`, I think I'll leave the code and comments as-is because I did not write it. I could have but it's some pretty tedious stuff! So I found a code written by someone else that worked and decided to use that instead.

Further to this, I've also just updated the content of `semver.sh` with a more complete library that I found: 
https://github.com/fsaintjacques/semver-tool